spanner/ddl/ast.go

```diff
@@ -494,6 +508,23 @@ func (k Foreignkey) PrintForeignKeyAlterTable(spannerSchema Schema, c Config, ta
 	return s
 }
 
+// PrintCheckConstraintTable formats the check constraints in SQL syntax.
+func PrintCheckConstraintTable(cks []CheckConstraint) string {
+	var builder strings.Builder
+
+	for _, col := range cks {
+		builder.WriteString(fmt.Sprintf("\tCONSTRAINT %s CHECK %s,\n", col.Name, col.Expr))
```

Comment:
There are check constraints possible without a name. This will fail in that case

Reply:
Can you please add a test case for this ?

Reply:
Added unit test